resource-managers/yarn/src/test/scala/org/apache/spark/network/yarn/YarnShuffleServiceSuite.scala

```diff
@@ -402,7 +402,9 @@ class YarnShuffleServiceSuite extends SparkFunSuite with Matchers with BeforeAnd
       "numRegisteredConnections",
       "openBlockRequestLatencyMillis",
       "registeredExecutorsSize",
-      "registerExecutorRequestLatencyMillis"
+      "registerExecutorRequestLatencyMillis",
+      "shuffle-server.usedDirectMemory",
+      "shuffle-server.usedHeapMemory"
```

Comment:
Apparently, the naming looks inconsistent from the others. Do you think we have a better way? For example, do we need `shuffle-server.` prefix?

Reply:
The metrics from `NettyMemoryMetrics` are both collected at server and client sides. That's why they have the module name (`shuffle-server` at server side) as prefix.

Reply:
Got it. Thanks.